src/pages/blog/2025-10-16-graphql-conf-2025-article-1/index.md

```diff
@@ -0,0 +1,196 @@
+
+Tom Houle showcased federation aware directives and policies in Grafbase Gateway that let platform teams centralize authorization without blocking subgraph autonomy.
+
+---
+
+## 🛠️ Tooling & DX
+
+![Jordan Eldredge](https://live.staticflickr.com/65535/54778865753_e076b321f5_c.jpg)
+
+### [Jordan Eldredge - Grats: Bringing Implementation-First GraphQL to TypeScript](https://graphql.org/conf/2025/schedule/e9075771b5513faaf06cca527e7a837d/?name=Grats:%20Bringing%20Implementation-First%20GraphQL%20to%20TypeScript%20-%20Jordan%20Eldredge,%20Meta)
+
+Jordan Eldredge previewed Meta's static analysis pipeline, showing how developers can build GraphQL servers in TypeScript without giving up type safety.
+
+---
+
+### [Mark Larah - Hacking the Federation Query Planner](https://graphql.org/conf/2025/schedule/806eaa5ecdc05b0c0f01165c7980b4a6/?name=Hacking%20the%20Federation%20Query%20Planner%20-%20Mark%20Larah,%20Yelp)
+
+Mark Larah unpacked a difficult concurrency edge case Yelp encountered and the long term spec fixes emerging through Composite Schemas.
+
+---
+
+### [Vanessa Johnson - What If GraphQL Knew Accessibility?](https://graphql.org/conf/2025/schedule/87264d07e7aaafb1811ba5ad451285a2/?name=Lightning%20Talk:%20What%20If%20GraphQL%20Knew%20Accessibility?%20-%20Vanessa%20Johnson,%20The%20New%20York%20Times)
+
+Vanessa Johnson proposed embedding accessibility metadata directly in schemas so tooling can generate inclusive experiences by default.
+
+---
+
+### [Fiona Huang - One API Definition To Rule Them All: Generating GraphQL Schemas From TypeSpec](https://graphql.org/conf/2025/schedule/0843f99870a32c08d091379a43d0c224/?name=One%20API%20Definition%20To%20Rule%20Them%20All:%20Generating%20GraphQL%20Schemas%20From%20TypeSpec%20-%20Fiona%20Huang,%20Pinterest)
+
+Fiona Huang announced Pinterest's TypeSpec GraphQL emitter that collapses REST and GraphQL definitions into a single source.
+
+---
+
+### [Eddy Nguyen - Building the Ideal GraphQL Server Workflow Featuring GraphQL Code Generator](https://graphql.org/conf/2025/schedule/0281a72e8e35f07c74a5815c42c64a02/?name=Sponsored%20Session:%20Building%20the%20Ideal%20GraphQL%20Server%20Workflow%20Featuring%20GraphQL%20Code%20Generator%20-%20Eddy%20Nguyen,%20SEEK%20&%20The%20Guild)
+
+Eddy Nguyen walked through a refreshed workflow powered by GraphQL Code Generator, highlighting stronger typing, fewer runtime errors, and lessons from a decade of community effort.
+
+---
+
+### [Derek Kuc - Efficient Semantic Comparison of GraphQL Queries](https://graphql.org/conf/2025/schedule/deac4044512d6d0a59c76aa712a777a4/?name=Lightning%20Talk:%20Efficient%20Semantic%20Comparison%20of%20GraphQL%20Queries%20-%20Derek%20Kuc,%20Apollo%20GraphQL)
+
+Derek Kuc unveiled static analysis that proves when two queries are equivalent and makes refactoring safer.
+
+---
```

Comment:
Maybe also somewhere we can link to the events and meetups page, if people want to keep track and join future events